app/views/users/new.html.erb

```diff
@@ -1,45 +1,47 @@
-<div id='clearance'>
-  
-<div class="float-left">
-  <div class="welcome-text-container">
-    <h2 class="welcome-title">Welcome to Acebook</h2>
-    <ul class="slides">
-      <li class="slide showing">"It's the greatest social network. It has the best posts.
-        The greatest photos. It's gonna be yee-uuuuge."</li>
-      <li class="slide">"My favourite social network, it is!"</li>
-      <li class="slide">"I met my dog walker on Acebook!"</li>
-      <li class="slide">"Hmm, interesting."</li>
-      <li class="slide">"Finally, another way to feel socially inadequate!"</li>
-    </ul>
-  </div>
-</div>
-
-<div class="sign-up-form">
-  <%= form_for @user, html: {class: 'new-user'} do |form| %>
+<body class='login-page'>
```

Comment:
A nice bit of learning for me that you can redefine the body class in this file like this.